bcipy/helpers/triggers.py

```diff
@@ -651,7 +651,10 @@ class TriggerType(Enum):
     TARGET = "target"
     FIXATION = "fixation"
     PROMPT = "prompt"
-    SYSTEM = "offset"
+    SYSTEM = "offset_correction"
+    CALIBRATION = "calib"
+    KEY_PRESS = "key_press"
```

Comment:
I'd be in favor of a more generic `EVENT` or similar and having the label describe what it is... key press, mouse click, etc. What do you think? It would constitute a change in the task as well. I'm happy to address that in another PR if you're okay with this approach.

Reply:
I added the `EVENT` prefix to the keypress for now, but I think this is not quite what you had in mind? Let's discuss in our next meeting and create a followup task for this.